tianshou/data/batch.py

```diff
@@ -29,6 +29,30 @@ def _is_batch_set(data: Any) -> bool:
     return False
 
 
+def is_scalar_value(value: Any) -> bool:
```

Comment:
I would rather emphasize that it is an internal method prefixing with `_`.

Reply:
And I not to simply call it `is_scalar` ?

Reply:
Fixed in 94f69b1 .